File: app/src/main/java/com/rachelleignacio/listbucket/presentation/presenters/RenameListFragmentPresenter.java
package com.rachelleignacio.listbucket.presentation.presenters;

import com.rachelleignacio.listbucket.domain.models.List;

/**
 * Created by rachelleignacio on 4/19/17.
 */

public interface RenameListFragmentPresenter {
    void renameList(List listToRename, String newListName);
}

File: app/src/main/kotlin/com/rachelleignacio/listbucket/presentation/presenters/RenameListFragmentPresenter.kt
package com.rachelleignacio.listbucket.presentation.presenters

import com.rachelleignacio.listbucket.domain.models.List

/**
 * Created by rachelleignacio on 8/23/17.
 */
interface RenameListFragmentPresenter {
    fun renameList(listToRename: List, newName: String)
}